src/views/ipv/page/prove-identity-no-photo-id.njk

```diff
@@ -10,19 +10,24 @@
 {% set errorHref = "#proveIdentityBankAccountOptionsForm" %}
 
 {% block content %}
-  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.proveIdentityBankAccount.header' | translate }}</h1>
-  <p class="govuk-body">{{ 'pages.proveIdentityBankAccount.content.paragraph1' | translate }}</p>
+  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.proveIdentityBankAccount.header' | translateWithContext(context) }}</h1>
+  {% if context != "NINumber" %}
```

Comment:
would expect the context key to be kebab-case ?

Reply:
Yes it will be Kebab case.